libs/designer-ui/src/lib/editor/string/index.tsx

```diff
@@ -16,6 +16,7 @@ export const StringEditor = ({
   initialValue,
   labelId,
   clearEditorOnTokenInsertion,
+  editorValueType,
```

Comment:
Can you call it valueType? editor prefix seems redundant for all editors.

Reply:
yeah sure, had did this because I didn't want to confuse it with valueType and keyType of dictionary editor, but I can make this change